### PR TITLE
Nokogiri version fix

### PR DESCRIPTION
**Pull request by sarangan12**
Tested with ruby 2.0, 2.1, 2.2, 2.3, 2.4 versions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ Information on Azure deployment models: [https://azure.microsoft.com/en-us/docum
 
 Note: x64 Ruby for Windows is known to have some compatibility issues.
 
+### Notice
+For ruby version >= 1.9.3 && < 2.2.0, please install compatible Nokogiri(version >= 1.6.8), otherwise the installation using old version of bundler or all version of rubygems will report failure.
```
Comment (vishrutshah) [inline README.md]: minor: feel free to use `and` instead of `&&` in the README.

Comment (sarangan12) [reply README.md]: Changed it

**Comment by sarangan12** [comment]
@vishrutshah Yes. I am planning to release it tomorrow. Bumping up will be seperate PR

**Comment by vishrutshah** [comment]
@sarangan12 I'd prefer to bump up version here only to avoid one more additional review and complete the work item here :)

**Comment by sarangan12** [comment]
@vishrutshah  np. Here you go....

**Comment by vishrutshah** [comment]
Cool. Thanks for bumping the version here to eliminate one more review :) :)

Let's fix the CI and see what's wrong with version `2.1.5` & `2.2.0`

**Comment by sarangan12** [comment]
@vishrutshah The error is happening even for simplest readme change https://travis-ci.org/Azure/azure-sdk-for-ruby/builds/272679479?utm_source=github_status&utm_medium=notification

I have created a travis issue: https://github.com/travis-ci/travis-ci/issues/8357

**Pull request merged**